### PR TITLE
Documentation for Browserstack local addon

### PR DESCRIPTION
**Pull request by anirudha13**
Added documentation for the BrowserStack add-on, detailing usage and example configurations.

### CODE REVIEW

File: user/browserstack.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+title: Using BrowserStack with Travis CI
+layout: en
+permalink: /user/browserstack/
+---
+Travis CI integrates with [BrowserStack](https://www.browserstack.com), a cross browser and real device 
+web-based testing platform. BrowserStack can be used for Live as well as automated testing through frameworks
```
Comment (plaindocs) [inline user/browserstack.md]: Does "Live" need to be capitalised?

File: user/browserstack.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+---
+title: Using BrowserStack with Travis CI
+layout: en
+permalink: /user/browserstack/
+---
+Travis CI integrates with [BrowserStack](https://www.browserstack.com), a cross browser and real device 
+web-based testing platform. BrowserStack can be used for Live as well as automated testing through frameworks 
+like Selenium, Karma and others.
+
+This add-on automatically sets up Local Testing which allows you to test your private servers, alongside public
```
Comment (plaindocs) [inline user/browserstack.md]: Also "local testing"?

File: user/browserstack.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+### Local Identifier
+
+A Local Identifier is a unique identifier for each Local connection when multiple Local connections are connected.
+The add-on will **ALWAYS** create a Local Identifier for each local connection that is created. If you are using the Selenium 
+testing framework, the Local Identifier must be added to the Selenium capabilities. 
+ 
+The Local Identifier is exposed as an environment variable `BROWSERSTACK_LOCAL_IDENTIFIER`. You can use it to set 
+the Selenium capability. See the following example which uses Ruby's [selenium-webdriver][browserstack-ruby-bindings]:
+
+    require 'rubygems'
```
Comment (plaindocs) [inline user/browserstack.md]: Could you code fence this so it gets highlighted? use 

    ```ruby
   ...
   ```

**Comment by anirudha13** [comment]
@plaindocs addressed comments in latest commit.

**Comment by plaindocs** [comment]
Awesome. Couple more picky comments:

* can you backtick `.travis.yml`
* left indent the fenced ruby code
* fence and left indent the yaml code 

Thanks!

**Comment by anirudha13** [comment]
@plaindocs addressed comments in latest commit.

**Comment by anirudha13** [comment]
@plaindocs any chance we can get this merged in today. The code PR has been merged, travis-ci/travis-build#702 .

File: user/browserstack.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    proxyPort: "Proxy server port"
+    proxyUser: "User to use when accessing proxy server"
+    proxyPass: "Password to use when accessing proxy server"
+```
+
+### More Options
+
+Some other options that are supported by the add on are,
+
+- **forcelocal**: If this is set to true then all URLs will be resolved via the Travis container that your build is running in.
```
Comment (plaindocs) [inline user/browserstack.md]: Can you explain this a little more?

**Comment by anirudha13** [comment]
@plaindocs when will this documentation get updated on the Travis docs site? We'd like to link to the page in our docs. 

Thanks.

**Pull request merged**